operator_ui/src/containers/Header.tsx

```diff
@@ -194,18 +193,35 @@ interface Props extends WithStyles<typeof styles> {
   url?: string
 }
 
-const Header = useHooks(
-  ({
-    authenticated,
-    classes,
-    fetchCount,
-    url,
-    drawerContainer,
-    onResize,
-    submitSignOut,
-  }: Props) => {
-    const [drawerOpen, setDrawerOpen] = useState(false)
-    const toggleDrawer = () => setDrawerOpen(!drawerOpen)
+interface State {
+  drawerOpen: boolean
+}
+
+class Header extends React.Component<Props, State> {
+  constructor(props: Props) {
```

Comment:
Is this needed? Isn't `props` type already inferred by the inherited class generic?

Reply:
ahhh yeah it is, good catch

Reply:
Oh, actually apparently not in strict mode.

https://fettblog.eu/typescript-react/components/#class-components

Reply:
duh, it's because the constructor is it's own constructor, feel free to resolve this